polaris.dme

```diff
@@ -1872,7 +1872,6 @@
 #include "code\modules\hydroponics\grown_predefined.dm"
 #include "code\modules\hydroponics\seed.dm"
 #include "code\modules\hydroponics\seed_controller.dm"
-#include "code\modules\hydroponics\seed_datums.dm"

```

Comment:
is there a reason we're executing seed_datums

Reply:
..God damnit branch swap

Reply:
I mean that code _does_ need to die in the fieriest pits of hell available...

Reply:
https://github.com/PolarisSS13/Polaris/pull/7475